### PR TITLE
Comment class branch

### LINKED ISSUE #2: Create Comment Model
Create Comment Model

### PR DESCRIPTION
**Pull request by barBen**
#2

### CODE REVIEW

File: DGN/Models/Comment.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+namespace DGN.Models
+{
+    public class Comment
+    {
+        // Primery key
+        public int Id { get; set; }
+
+        [Required]
+        [StringLenght(20), MinimumLenght = 5]
```
Comment (origranot) [inline DGN/Models/Comment.cs]: String LENGHHHTTT

File: DGN/Models/Comment.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        [RegularExpretion("^[A-Z]*$")]
+        public string Title { get; set; }
```
Comment (bsuro10) [inline DGN/Models/Comment.cs]: We don't need title property

File: DGN/Models/Comment.cs [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+        [StringLenght(200), MinimumLenght = 5]
```
Comment (bsuro10) [inline DGN/Models/Comment.cs]: StringLength*

File: DGN/Models/Comment.cs [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+        public string Body { get; set; }
+
+        // Shouldnt display in the GET form
+        [ForeignKey("User")]
+        public User CommentWriter { get; set; }
```
Comment (bsuro10) [inline DGN/Models/Comment.cs]: [Required]
[ForeignKey("User")]
public int AuthorId { get; set; }

[Required]
public User Author { get; set; }

File: DGN/Models/Comment.cs [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+        // Shouldnt display in the GET form
+        [ForeignKey("Article")]
+        public Article RelatedArticle { get; set; }
```
Comment (bsuro10) [inline DGN/Models/Comment.cs:34]: [Required]
[ForeignKey("Article")]
public int ArticleId { get; set; }

[Required]
public Article Article { get; set; }

**Review by bsuro10 — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

**Review by ghost — APPROVED** [review]

File: DGN/Models/Comment.cs [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+namespace DGN.Models
+{
+    public class Comment
+    {
+        // Primery key
+        public int Id { get; set; }
+
+        [StringLength(200), MinimumLength = 5]
+        public string Body { get; set; }
```
Comment (bsuro10) [inline DGN/Models/Comment.cs:18]: Need to add `[DataType(DataType.MultilineText)]` and `[Required]`

**Review by ghost — DISMISSED** [review]

**Review by bsuro10 — CHANGES_REQUESTED** [review]

**Review by bsuro10 — APPROVED** [review]

**Review by origranot — APPROVED** [review]

**Pull request merged**